tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/AdoNet/Sqlite/SqliteDefinitions.cs

```diff
@@ -62,9 +62,9 @@
 
 [assembly: AdoNetClientInstrumentMethods(
     AssemblyName = "Microsoft.Data.Sqlite",
-    TypeName = "Microsoft.Data.Sqlite.SQLiteDataReader",
+    TypeName = "Microsoft.Data.Sqlite.SqliteDataReader",
     MinimumVersion = "2.0.0",
-    MaximumVersion = "8.*.*",
+    MaximumVersion = "9.*.*",
     IntegrationName = nameof(IntegrationId.SqlClient),
     DataReaderType = "System.Data.SQLite.SQLiteDataReader",
```

Comment:
not from this PR, but, is this correct 🤔 It should definitely be a `System.Data.SQLite.SQLiteDataReader` type even though this is the `Microsoft.Data.sqlite` package? 🤨

Reply:
Yea that's should definitely be that. I thin kon this case it did work because they inherit of the same base?
I updated it in ba0a87265bbc693f7fa41653310db0970f8ea8a6